fair/__init__.py

```diff
@@ -2,11 +2,11 @@
 # @Author: zebedeen
```

Comment:
if you want, change this to you (or delete these header strings altogether, I think they're obsolete...)

Reply:
Agreed, looks like something automatically inserted with a particular text editor.